### ISSUE TITLE
waveglow model

### ISSUE THREAD

**Issue by yondu22** [NVIDIA/tacotron2#262]
If I train a new tacotron with my own data, do I need to train a new waveglow model or can I used the pretrained one?

**Comment by hadaev8** [comment]
Seems pretrained waveglow works fine even for non english data

**Comment by yondu22** [comment]
it also works for different voices, kids and adults. Why do we need to even train our own waveglow model?

**Comment by rafaelvalle** [comment]
@yondu22 fine-tuning waveglow to your own data *can* produce better results.

**Comment by hadaev8** [comment]
@rafaelvalle what kind of learning rate should be used for fine-tuning?

**Comment by rafaelvalle** [comment]
Basically drop the learning rate if the loss goes up or if after some time it does not decrease.

**Comment by rafaelvalle** [comment]
Closing due to inactivity.

**Comment by patrick-g-zhang** [comment]
seems waveglow like a universal vocoder.
